### DISCUSSION TITLE
Limit the number of characters in a cell

### DISCUSSION THREAD

**Discussion by rmrbytes** [iddan/react-spreadsheet#273] [Q&A] [answered]
Is there a way we can control the number of text input in a cell? Thanks

**Comment by iddan** [comment, marked as answer]
Hey, you can bypass the DataEditor component https://iddan.github.io/react-spreadsheet/docs/api#type-declaration

**Comment by nappalm** [comment]
@iddan page not found

**Comment by iddan** [comment]
Updated link: https://iddan.github.io/react-spreadsheet/api#Props
